scripts/init.sh

```diff
@@ -51,7 +51,7 @@ sudo cp files/system/issue.net /etc/
 # Pin mapping configuration
 echo "{}" | sudo tee /etc/hobby-hub/pin_mapping.json # creates empty, but valid JSON file
 chmod og=rw /etc/hobby-hub/pin_mapping.json
-cp files/src/pin_manager.py /etc/hobby-hub/ # pin manager itself
```

Comment:
is this file not being used anymore?

Reply:
yeah, remember we talked about this with Rock? that pin_manager.py files should be embedded in the hobby-hub package no?

Reply:
@Ivendrakeo